File: server/src/main/java/org/cleanas2/bus/SaveMdnMsg.java
package org.cleanas2.bus;

import org.cleanas2.message.IncomingSyncMdn;

/**
 * @author Andrew Backer {@literal awbacker@gmail.com / andrew.backer@powere2e.com}
 */
public class SaveMdnMsg extends MessageBase {
    public final IncomingSyncMdn mdn;

    public SaveMdnMsg(IncomingSyncMdn as2mdn) {
        mdn = as2mdn;
    }
}

File: server/src/main/java/org/cleanas2/bus/SaveMdnMsg.kt
package org.cleanas2.bus

import org.cleanas2.message.IncomingSyncMdn

/**
 * @author Andrew Backer awbacker@gmail.com / andrew.backer@powere2e.com
 */
class SaveMdnMsg(val mdn: IncomingSyncMdn) : MessageBase()
